.drone.yml

```diff
@@ -1,16 +1,40 @@
 ---
 kind: pipeline
-name: default
+name: testing
 
 platform:
   os: linux
   arch: amd64
 
 steps:
+- name: vet
+  pull: always
+  image: golang:1.11
+  commands:
+  - go vet ./...
+  environment:
+    GO111MODULE: on
+  volumes:
+  - name: gopath
+    path: /go
+
 - name: test
+  pull: always
```

Comment:
I actually prefer not to use `pull: always` unless necessary because it makes `drone exec` run slow locally, since it has to check dockerhub for updates. Also requires an internet connection

Reply:
Ok, I can just drop that.